jobs/ml-ops/container/inference-api/Dockerfile

```
@@ -0,0 +1,10 @@
+FROM python:3.10.12
```

nit: If there's no specific reason to use python3.10, we can bump it to 3.12, there are a lot of performance improvement ^^

```suggestion
FROM python:3.12-slim-bookworm
```